tox.ini

```diff
@@ -0,0 +1,29 @@
+[testenv]
+deps =
+    coverage >= 4.5.1
+    responses >= 0.21.0
+    unittest-parallel >= 1.5.0
+commands =
+    unittest-parallel -t . -s tests --coverage --coverage-rcfile .coveragerc
+
+# The system-provided libxml2 on MacOS is typically outdated and this can lead to lxml parsing issues
+# Using PyPi-provided binary wheels instead resolves this
+# We are affected by https://bugs.launchpad.net/lxml/+bug/1949271 in test_wild_mode when using system-provided libxml2 on MacOS
+platform =
+    py-darwin: darwin
+install_command =
+    py-darwin: python -m pip install --only-binary=lxml {opts} {packages}
+
+[testenv:lint]
+skip_install = true
+deps =
+    black >= 22.3.0
+    flake8 >= 3.8.3
+    flake8-printf-formatting >= 1.1.0
+    mypy >= 0.971
+    types-beautifulsoup4 >= 4.11.6
+    types-requests >= 2.28.10
+commands =
+    black --check .
+    flake8 --count .
```

Comment:
Note: this is a like-for-like migration of the existing `flake8` behaviour (count only).  To help people out in future though, we could/should probably run it without arguments so that it produces output when it detects problems.